eng/build.ps1

```diff
@@ -12,7 +12,7 @@ function TestUsingNPM([string] $testPath) {
 
 try {
     # invoke regular build/test script
-    . (Join-Path $PSScriptRoot "common\build.ps1") @args
+    . (Join-Path $PSScriptRoot "common\build.ps1") "-projects $PSScriptRoot\..\dotnet-interactive.sln;$PSScriptRoot\..\dotnet-try.sln"
```

Comment:
You still have to pass through `@args`.

Reply:
Also, you _might_ have access to the variable `$RepoRoot` so you don't need `$PSScriptRoot\..`.

Reply:
then why the line below in test like `TestUsingNPM "$PSScriptRoot\..\Microsoft.DotNet.Try.Client"`